tests/mgbench/benchmark.py

```diff
@@ -455,15 +379,15 @@ def get_query_cache_count(vendor, client, func, config_key):
                 count = count * 10
         vendor.stop("cache")
 
-        # Lower bound for count
-        if count < 20:
-            count = 20
+        count_lower_bound = 30
```

Comment:
Move this as a top-level constant with caps (Python standard for constants, `COUNT_LOWER_BOUND`)